### ISSUE TITLE
Calamares 3.3.0~alpha5 fails to build on Debian: "error: no match for ‘operator>'"

### ISSUE THREAD

**Issue by highvoltage** [calamares/calamares#2235]
While building calamares 3.3.0~alpha5 on Debian, I ran into the following compilation error:

```
[ 69%] Building CXX object src/modules/partition/CMakeFiles/calamares_viewmodule_partition.dir/Config.cpp.o
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp: In function ‘void fillGSConfigurationEFI(Calamares::GlobalStorage*, const QVariantMap&)’:
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp:323:32: error: no match for ‘operator>’ (operand types are ‘Calamares::Partition::PartitionSize’ and ‘qint64’ {aka ‘long long int’})
  323 |                 if ( part_size > PartUtils::efiFilesystemRecommendedSize() )
      |                      ~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                      |                                                  |
      |                      Calamares::Partition::PartitionSize                qint64 {aka long long int}
In file included from /data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp:16:
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/libcalamares/partition/PartitionSize.h:61:10: note: candidate: ‘bool Calamares::Partition::PartitionSize::operator>(const Calamares::Partition::PartitionSize&) const’
   61 |     bool operator>( const PartitionSize& other ) const;
      |          ^~~~~~~~
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/libcalamares/partition/PartitionSize.h:61:42: note:   no known conversion for argument 1 from ‘qint64’ {aka ‘long long int’} to ‘const Calamares::Partition::PartitionSize&’
   61 |     bool operator>( const PartitionSize& other ) const;
      |                     ~~~~~~~~~~~~~~~~~~~~~^~~~~
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qhashfunctions.h:45,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/qlist.h:47,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/qobject.h:49,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/QObject:1,
                 from /usr/include/kpmcore/core/partitionnode.h:14,
                 from /usr/include/kpmcore/core/partition.h:15,
                 from /data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.h:15,
                 from /data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp:10:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qpair.h:130:41: note: candidate: ‘template<class T1, class T2> constexpr bool operator>(const QPair<T1, T2>&, const QPair<T1, T2>&)’
  130 | Q_DECL_CONSTEXPR Q_INLINE_TEMPLATE bool operator>(const QPair<T1, T2> &p1, const QPair<T1, T2> &p2)
      |                                         ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qpair.h:130:41: note:   template argument deduction/substitution failed:
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp:323:74: note:   ‘Calamares::Partition::PartitionSize’ is not derived from ‘const QPair<T1, T2>’
  323 |                 if ( part_size > PartUtils::efiFilesystemRecommendedSize() )
      |                                                                          ^
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qlist.h:48:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvector.h:1096:13: note: candidate: ‘template<class T> bool operator>(const QVector<T>&, const QVector<T>&)’
 1096 | inline bool operator>(const QVector<T> &lhs, const QVector<T> &rhs)
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvector.h:1096:13: note:   template argument deduction/substitution failed:
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp:323:74: note:   ‘Calamares::Partition::PartitionSize’ is not derived from ‘const QVector<T>’
  323 |                 if ( part_size > PartUtils::efiFilesystemRecommendedSize() )
      |                                                                          ^
/usr/include/x86_64-linux-gnu/qt5/QtCore/qlist.h:1173:13: note: candidate: ‘template<class T> bool operator>(const QList<T>&, const QList<T>&)’
 1173 | inline bool operator>(const QList<T> &lhs, const QList<T> &rhs)
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qlist.h:1173:13: note:   template argument deduction/substitution failed:
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp:323:74: note:   ‘Calamares::Partition::PartitionSize’ is not derived from ‘const QList<T>’
  323 |                 if ( part_size > PartUtils::efiFilesystemRecommendedSize() )
      |                                                                          ^
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qmetatype.h:48,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/qobject.h:54:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvarlengtharray.h:588:13: note: candidate: ‘template<class T, int Prealloc1, int Prealloc2> bool operator>(const QVarLengthArray<T, Prealloc1>&, const QVarLengthArray<T, Prealloc2>&)’
  588 | inline bool operator>(const QVarLengthArray<T, Prealloc1> &lhs, const QVarLengthArray<T, Prealloc2> &rhs)
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvarlengtharray.h:588:13: note:   template argument deduction/substitution failed:
/data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.cpp:323:74: note:   ‘Calamares::Partition::PartitionSize’ is not derived from ‘const QVarLengthArray<T, Prealloc1>’
  323 |                 if ( part_size > PartUtils::efiFilesystemRecommendedSize() )
      |                                                                          ^
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:49,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/QString:1,
                 from /data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/libcalamares/utils/NamedEnum.h:26,
                 from /data/jonathan/devel/debian/kde-team/calamares/calamares/build/src/modules/partition/Config.h:13:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:66:30: note: candidate: ‘constexpr bool operator>(char, QLatin1Char)’
   66 | Q_DECL_CONSTEXPR inline bool operator> (char lhs, QLatin1Char rhs) noexcept { return lhs >  rhs.toLatin1(); }
      |                              ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:66:46: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘char’
   66 | Q_DECL_CONSTEXPR inline bool operator> (char lhs, QLatin1Char rhs) noexcept { return lhs >  rhs.toLatin1(); }
      |                                         ~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:73:30: note: candidate: ‘constexpr bool operator>(QLatin1Char, char)’
   73 | Q_DECL_CONSTEXPR inline bool operator> (QLatin1Char lhs, char rhs) noexcept { return lhs.toLatin1() >  rhs; }
      |                              ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:73:53: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QLatin1Char’
   73 | Q_DECL_CONSTEXPR inline bool operator> (QLatin1Char lhs, char rhs) noexcept { return lhs.toLatin1() >  rhs; }
      |                                         ~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:642:30: note: candidate: ‘constexpr bool operator>(QChar, QChar)’
  642 | Q_DECL_CONSTEXPR inline bool operator> (QChar c1, QChar c2) noexcept { return  operator< (c2, c1); }
      |                              ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:642:47: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QChar’
  642 | Q_DECL_CONSTEXPR inline bool operator> (QChar c1, QChar c2) noexcept { return  operator< (c2, c1); }
      |                                         ~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:653:30: note: candidate: ‘constexpr bool operator>(QChar, std::nullptr_t)’
  653 | Q_DECL_CONSTEXPR inline bool operator> (QChar lhs, std::nullptr_t) noexcept { return  operator< (nullptr, lhs); }
      |                              ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:653:47: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QChar’
  653 | Q_DECL_CONSTEXPR inline bool operator> (QChar lhs, std::nullptr_t) noexcept { return  operator< (nullptr, lhs); }
      |                                         ~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:658:30: note: candidate: ‘constexpr bool operator>(std::nullptr_t, QChar)’
  658 | Q_DECL_CONSTEXPR inline bool operator> (std::nullptr_t, QChar rhs) noexcept { return  operator< (rhs, nullptr); }
      |                              ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qchar.h:658:41: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘std::nullptr_t’
  658 | Q_DECL_CONSTEXPR inline bool operator> (std::nullptr_t, QChar rhs) noexcept { return  operator< (rhs, nullptr); }
      |                                         ^~~~~~~~~~~~~~
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:50:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qbytearray.h:708:13: note: candidate: ‘bool operator>(const QByteArray&, const QByteArray&)’
  708 | inline bool operator>(const QByteArray &a1, const QByteArray &a2) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qbytearray.h:708:41: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QByteArray&’
  708 | inline bool operator>(const QByteArray &a1, const QByteArray &a2) noexcept
      |                       ~~~~~~~~~~~~~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qbytearray.h:710:13: note: candidate: ‘bool operator>(const QByteArray&, const char*)’
  710 | inline bool operator>(const QByteArray &a1, const char *a2) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qbytearray.h:710:41: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QByteArray&’
  710 | inline bool operator>(const QByteArray &a1, const char *a2) noexcept
      |                       ~~~~~~~~~~~~~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qbytearray.h:712:13: note: candidate: ‘bool operator>(const char*, const QByteArray&)’
  712 | inline bool operator>(const char *a1, const QByteArray &a2) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qbytearray.h:712:35: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const char*’
  712 | inline bool operator>(const char *a1, const QByteArray &a2) noexcept
      |                       ~~~~~~~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1396:13: note: candidate: ‘bool operator>(QLatin1String, QLatin1String)’
 1396 | inline bool operator>(QLatin1String s1, QLatin1String s2) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1396:37: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QLatin1String’
 1396 | inline bool operator>(QLatin1String s1, QLatin1String s2) noexcept
      |                       ~~~~~~~~~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1436:32: note: candidate: ‘bool operator>(const char*, const QString&)’
 1436 | inline QT_ASCII_CAST_WARN bool operator>(const char *s1, const QString &s2)
      |                                ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1436:54: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const char*’
 1436 | inline QT_ASCII_CAST_WARN bool operator>(const char *s1, const QString &s2)
      |                                          ~~~~~~~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1449:32: note: candidate: ‘bool operator>(const char*, QLatin1String)’
 1449 | inline QT_ASCII_CAST_WARN bool operator>(const char *s1, QLatin1String s2)
      |                                ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1449:54: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const char*’
 1449 | inline QT_ASCII_CAST_WARN bool operator>(const char *s1, QLatin1String s2)
      |                                          ~~~~~~~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1825:13: note: candidate: ‘bool operator>(const QStringRef&, const QStringRef&)’
 1825 | inline bool operator>(const QStringRef &s1, const QStringRef &s2) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1825:41: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QStringRef&’
 1825 | inline bool operator>(const QStringRef &s1, const QStringRef &s2) noexcept
      |                       ~~~~~~~~~~~~~~~~~~^~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1836:13: note: candidate: ‘bool operator>(const QString&, const QStringRef&)’
 1836 | inline bool operator> (const QString &lhs, const QStringRef &rhs) noexcept { return lhs.compare(rhs) >  0; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1836:39: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QString&’
 1836 | inline bool operator> (const QString &lhs, const QStringRef &rhs) noexcept { return lhs.compare(rhs) >  0; }
      |                        ~~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1843:13: note: candidate: ‘bool operator>(const QStringRef&, const QString&)’
 1843 | inline bool operator> (const QStringRef &lhs, const QString &rhs) noexcept { return rhs <  lhs; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1843:42: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QStringRef&’
 1843 | inline bool operator> (const QStringRef &lhs, const QString &rhs) noexcept { return rhs <  lhs; }
      |                        ~~~~~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1872:13: note: candidate: ‘bool operator>(QLatin1String, const QStringRef&)’
 1872 | inline bool operator> (QLatin1String lhs, const QStringRef &rhs) noexcept { return rhs.compare(lhs) <  0; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1872:38: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QLatin1String’
 1872 | inline bool operator> (QLatin1String lhs, const QStringRef &rhs) noexcept { return rhs.compare(lhs) <  0; }
      |                        ~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1879:13: note: candidate: ‘bool operator>(const QStringRef&, QLatin1String)’
 1879 | inline bool operator> (const QStringRef &lhs, QLatin1String rhs) noexcept { return rhs <  lhs; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1879:42: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QStringRef&’
 1879 | inline bool operator> (const QStringRef &lhs, QLatin1String rhs) noexcept { return rhs <  lhs; }
      |                        ~~~~~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1888:13: note: candidate: ‘bool operator>(QChar, const QString&)’
 1888 | inline bool operator> (QChar lhs, const QString &rhs) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1888:30: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QChar’
 1888 | inline bool operator> (QChar lhs, const QString &rhs) noexcept
      |                        ~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1898:13: note: candidate: ‘bool operator>(const QString&, QChar)’
 1898 | inline bool operator> (const QString &lhs, QChar rhs) noexcept { return   rhs <  lhs; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1898:39: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QString&’
 1898 | inline bool operator> (const QString &lhs, QChar rhs) noexcept { return   rhs <  lhs; }
      |                        ~~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1907:13: note: candidate: ‘bool operator>(QChar, const QStringRef&)’
 1907 | inline bool operator> (QChar lhs, const QStringRef &rhs) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1907:30: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QChar’
 1907 | inline bool operator> (QChar lhs, const QStringRef &rhs) noexcept
      |                        ~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1917:13: note: candidate: ‘bool operator>(const QStringRef&, QChar)’
 1917 | inline bool operator> (const QStringRef &lhs, QChar rhs) noexcept { return   rhs <  lhs; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1917:42: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QStringRef&’
 1917 | inline bool operator> (const QStringRef &lhs, QChar rhs) noexcept { return   rhs <  lhs; }
      |                        ~~~~~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1926:13: note: candidate: ‘bool operator>(QChar, QLatin1String)’
 1926 | inline bool operator> (QChar lhs, QLatin1String rhs) noexcept
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1926:30: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QChar’
 1926 | inline bool operator> (QChar lhs, QLatin1String rhs) noexcept
      |                        ~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1936:13: note: candidate: ‘bool operator>(QLatin1String, QChar)’
 1936 | inline bool operator> (QLatin1String lhs, QChar rhs) noexcept { return   rhs <  lhs; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1936:38: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QLatin1String’
 1936 | inline bool operator> (QLatin1String lhs, QChar rhs) noexcept { return   rhs <  lhs; }
      |                        ~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1945:13: note: candidate: ‘bool operator>(QStringView, QStringView)’
 1945 | inline bool operator> (QStringView lhs, QStringView rhs) noexcept { return QtPrivate::compareStrings(lhs, rhs) >  0; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1945:36: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QStringView’
 1945 | inline bool operator> (QStringView lhs, QStringView rhs) noexcept { return QtPrivate::compareStrings(lhs, rhs) >  0; }
      |                        ~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1953:13: note: candidate: ‘bool operator>(QStringView, QChar)’
 1953 | inline bool operator> (QStringView lhs, QChar rhs) noexcept { return lhs >  QStringView(&rhs, 1); }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1953:36: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QStringView’
 1953 | inline bool operator> (QStringView lhs, QChar rhs) noexcept { return lhs >  QStringView(&rhs, 1); }
      |                        ~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1960:13: note: candidate: ‘bool operator>(QChar, QStringView)’
 1960 | inline bool operator> (QChar lhs, QStringView rhs) noexcept { return QStringView(&lhs, 1) >  rhs; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1960:30: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QChar’
 1960 | inline bool operator> (QChar lhs, QStringView rhs) noexcept { return QStringView(&lhs, 1) >  rhs; }
      |                        ~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1968:13: note: candidate: ‘bool operator>(QStringView, QLatin1String)’
 1968 | inline bool operator> (QStringView lhs, QLatin1String rhs) noexcept { return QtPrivate::compareStrings(lhs, rhs) >  0; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1968:36: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QStringView’
 1968 | inline bool operator> (QStringView lhs, QLatin1String rhs) noexcept { return QtPrivate::compareStrings(lhs, rhs) >  0; }
      |                        ~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1975:13: note: candidate: ‘bool operator>(QLatin1String, QStringView)’
 1975 | inline bool operator> (QLatin1String lhs, QStringView rhs) noexcept { return QtPrivate::compareStrings(lhs, rhs) >  0; }
      |             ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1975:38: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘QLatin1String’
 1975 | inline bool operator> (QLatin1String lhs, QStringView rhs) noexcept { return QtPrivate::compareStrings(lhs, rhs) >  0; }
      |                        ~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1983:32: note: candidate: ‘bool operator>(const QStringRef&, const QByteArray&)’
 1983 | inline QT_ASCII_CAST_WARN bool operator> (const QStringRef &lhs, const QByteArray &rhs) { return lhs.compare(rhs) >  0; }
      |                                ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1983:61: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QStringRef&’
 1983 | inline QT_ASCII_CAST_WARN bool operator> (const QStringRef &lhs, const QByteArray &rhs) { return lhs.compare(rhs) >  0; }
      |                                           ~~~~~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1990:32: note: candidate: ‘bool operator>(const QByteArray&, const QStringRef&)’
 1990 | inline QT_ASCII_CAST_WARN bool operator> (const QByteArray &lhs, const QStringRef &rhs) { return rhs.compare(lhs) <  0; }
      |                                ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:1990:61: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const QByteArray&’
 1990 | inline QT_ASCII_CAST_WARN bool operator> (const QByteArray &lhs, const QStringRef &rhs) { return rhs.compare(lhs) <  0; }
      |                                           ~~~~~~~~~~~~~~~~~~^~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:2016:32: note: candidate: ‘bool operator>(const char*, const QStringRef&)’
 2016 | inline QT_ASCII_CAST_WARN bool operator>(const char *s1, const QStringRef &s2)
      |                                ^~~~~~~~
/usr/include/x86_64-linux-gnu/qt5/QtCore/qstring.h:2016:54: note:   no known conversion for argument 1 from ‘Calamares::Partition::PartitionSize’ to ‘const char*’
 2016 | inline QT_ASCII_CAST_WARN bool operator>(const char *s1, const QStringRef &s2)
      |                                          ~~~~~~~~~~~~^~
make[2]: *** [src/modules/partition/CMakeFiles/calamares_viewmodule_partition.dir/build.make:90: src/modules/partition/CMakeFiles/calamares_viewmodule_partition.dir/Config.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:7040: src/modules/partition/CMakeFiles/calamares_viewmodule_partition.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

I haven't looked into it much, but it doesn't seem to be a Debian specific issue. I think I caught all the changes from recent Calamares news events, so it doesn't seem like a misconfiguration. Thanks!

**Comment by demmm** [comment]
Please post the build settings used (CMake calls).

**Comment by highvoltage** [comment]
Hey @demmm - I'm not that advanced with cmake, in the clean upstream source, I did:

```
mkdir build
cd build
cmake ..
make
```

**Comment by philmmanjaro** [comment]
Same happens on Manjaro too:  [3.3.0alpha5.r11671.bb99e4790-1.log](https://github.com/calamares/calamares/files/13370340/3.3.0alpha5.r11671.bb99e4790-1.log)

**Comment by philmmanjaro** [comment]
Seems also happen on the CI: https://github.com/calamares/calamares/actions/runs/6855642761/job/18641136428#step:5:1461 Try with the latest commit.

**Comment by adriaandegroot** [comment]
This is known, and was fixed after -alpha5, which will lead to a -beta1 Real Soon

**Comment by highvoltage** [comment]
Ah, it would probably be good if the issue is known to people who implement the software too, perhaps it would be a good idea to mention it in the news note for alpha5.

**Comment by adriaandegroot** [comment]
Fixed by 8868ae956ac2c620082d6fae7ac251c92ce91c9a, -alpha6 out shortly.
